recipes-kernel/linux/files/wm8960.cfg

```diff
@@ -0,0 +1,2 @@
+CONFIG_I2C_BCM2835=y
+CONFIG_SND_SOC_WM8960=m
```

Comment:
should it be a module ?

Reply:
Could you point to a module example to clarify?

Reply:
@tewarid kraj is asking if this kconfig should really be a module and not built-in. If it is a module, we need to make sure that the kernel module ends up in the rootfs as well.

Reply:
@agherzan Thanks for the clarification :)

I'll make it built-in.